getMedia.js

```diff
@@ -154,6 +168,35 @@ const resolvers = {
         next(providers, mediaURL, options, firstErr || err, callback)
       })
     })
+  },
+
+  Twitch: (providers, mediaURL, options, payload, firstErr, callback) => {
+    resolvers._channel(providers, mediaURL, options, underscore.extend({
+      _channel: {
+        providerName: 'twitch',
+        param1: 2,
```

Comment:
are we using this one?

Reply:
yes. that's the point of the PR. i refactored so the twitch and youtube code could maximally reuse code.

Reply:
where do we use it? in other library? I can't find it in this PR

Reply:
take a look at media/provider.json!